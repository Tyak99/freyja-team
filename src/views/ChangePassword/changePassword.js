import React from 'react';
// eslint-disable-next-line import/no-named-as-default
import ChangePassword from '../../components/ChangePassword/ChangePassword';

const ResetPassword = () => (
  <ChangePassword />
);


export default ResetPassword;
