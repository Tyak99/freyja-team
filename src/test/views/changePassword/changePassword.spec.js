import React from 'react';
import { shallow } from 'enzyme';
import ChangePassword from '../../../views/ChangePassword/changePassword';


describe('reser password page', () => {
  it('should render correctly Reset password page', () => {
    const ResetPassword = shallow(<ChangePassword />);
    expect(ResetPassword).toMatchSnapshot();
  });
});
