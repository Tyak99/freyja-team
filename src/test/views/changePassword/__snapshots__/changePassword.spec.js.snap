// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reser password page should render correctly Reset password page 1`] = `ShallowWrapper {}`;
